src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/ServiceRoleMapping.java

```diff
@@ -19,7 +24,7 @@
     String serivceId;
 
     @MappingField(columnName = "Roles")
-    @NotEmpty
+    @NotNull
```

Comment:
I think we need change primitive to Integer wrapper else @Notnull never catch up.

Reply:
Sure